image_services/rtc_services/nasa_disasters/rgb.json

```
@@ -27,6 +27,8 @@
     "ItemInfo/Title": "ASF S1 RGB",
     "ItemInfo/Credits": "RGB Decomposition products processed by ASF DAAC HyP3 using GAMMA software. Contains modified Copernicus Sentinel data, processed by ESA.",
     "ItemInfo/Description": "Sentinel-1 RGB Decomposition of RTC VV and VH imagery, processed by ASF. Blue areas have low returns in VV and VH (smooth surfaces such as calm water, but also frozen/crusted soil or dry sand), green areas have high returns in VH (volume scatterers such as vegetation or some types of snow/ice), and red areas have relatively high VV returns and relatively low VH returns (such as urban or sparsely vegetated areas).",
-    "ItemInfo/Snippet": "RGB Decomposition of Sentinel-1 Radiometric Terrain Corrected (RTC) VV and VH imagery, processed by ASF."
+    "ItemInfo/Snippet": "RGB Decomposition of Sentinel-1 Radiometric Terrain Corrected (RTC) VV and VH imagery, processed by ASF.",
+    "Configurations/SVCConfiguration/Definition/Extensions/SVCExtension/Props/PropertyArray/PropertySetProperty[Key='title']/Value": "ASF_S1_RGB_WMS",
```

Looks like a service definition draft could have multiple extensions, so probably safer to search for the WMSServer extension specifically:
```suggestion
    "Configurations/SVCConfiguration/Definition/Extensions/SVCExtension[TypeName='WMSServer]/Props/PropertyArray/PropertySetProperty[Key='title']/Value": "ASF_S1_RGB_WMS",
```